<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-10</y>
      <width>221</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ADD8E6;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #87CEFA;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="Sante">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>901</width>
      <height>551</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ajouter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ajouter&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab{
    color:white;
	font: 75 10pt &quot;Source Code Pro&quot;;
	font: 87 8pt &quot;Nexa Rust Sans Black 01&quot;;
    background-color:rgb(245, 245,245);
background:  rgb(245, 129, 233);
border-top-left-radius:10px;
border-top-right-radius:10px;
min-width:120px;
height:30px;

}
QTabBar::tab::hover
{
color:white;

background:#b20837;
border-top-left-radius:10px;
border-top-right-radius:10px;
}

QTabBar::tab:selected
{
color:white;

background:#b20837;
border-top-left-radius:10px;
border-top-right-radius:10px;
}</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Equipements</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-2</y>
        <width>901</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab{

height:28px;

}</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>691</width>
          <height>411</height>
         </rect>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QLineEdit" name="nom_line">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>60</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="prix_l">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>120</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="marque_l">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>200</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="caracteristique_l">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>290</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="etat_l">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>250</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="date_l">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>100</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="aj">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>360</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="id_line">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>20</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Consulter</string>
       </attribute>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>60</y>
          <width>771</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Reparateurs</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_4">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-2</y>
        <width>901</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab{

height:28px;

}</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Consulter</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_11">
     <attribute name="title">
      <string>Sant√©</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>721</width>
        <height>441</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>160</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>120</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Prix</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>155</y>
          <width>55</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Niveau</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>160</y>
          <width>118</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>120</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>411</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>100</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Prix</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>110</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>240</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>150</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Niveau</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_profil">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>130</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color:white;
	font: 75 10pt &quot;Source Code Pro&quot;;
	font: 87 8pt &quot;Nexa Rust Sans Black 01&quot;;
    background-color:rgb(39, 39,39);
background: transparent;
border-radius:10px;
}
QPushButton:hover
{
color:white;

background:#b20837;
border-radius:10px;
}

QPushButton:focus
{
color:white;

background:#b20837;
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string> Profil                     </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/img/img/154347018_1093458397748791_4413400449242006441_n.png</normaloff>:/img/img/154347018_1093458397748791_4413400449242006441_n.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_client">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>230</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color:white;
	font: 75 10pt &quot;Source Code Pro&quot;;
	font: 87 8pt &quot;Nexa Rust Sans Black 01&quot;;
    background-color:rgb(39, 39,39);
background: transparent;
border-radius:10px;
}
QPushButton:hover
{
color:white;

background:#b20837;
border-radius:10px;
}

QPushButton:focus
{
color:white;

background:#b20837;
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>  Clients/Commande </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/img/img/Untitled design (21).png</normaloff>:/img/img/Untitled design (21).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_produit">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>280</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color:white;
	font: 75 10pt &quot;Source Code Pro&quot;;
	font: 87 8pt &quot;Nexa Rust Sans Black 01&quot;;
    background-color:rgb(39, 39,39);
background: transparent;
border-radius:10px;
}
QPushButton:hover
{
color:white;

background:#b20837;
border-radius:10px;
}

QPushButton:focus
{
color:white;

background:#b20837;
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>  Produits                  </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/img/img/Untitled design (22).png</normaloff>:/img/img/Untitled design (22).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>39</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/img/154067405_1632533030284127_9027939143200895570_n.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="button_equipement">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color:white;
	font: 75 10pt &quot;Source Code Pro&quot;;
	font: 87 8pt &quot;Nexa Rust Sans Black 01&quot;;
    background-color:rgb(39, 39,39);
background: transparent;
border-radius:10px;
}
QPushButton:hover
{
color:white;

background:#b20837;
border-radius:10px;
}

QPushButton:focus
{
color:white;

background:#b20837;
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string> Equipements            </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/img/img/Untitled design (23).png</normaloff>:/img/img/Untitled design (23).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_fournisseurs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color:white;
	font: 75 10pt &quot;Source Code Pro&quot;;
	font: 87 8pt &quot;Nexa Rust Sans Black 01&quot;;
    background-color:rgb(39, 39,39);
background: transparent;
border-radius:10px;
}
QPushButton:hover
{
color:white;

background:#b20837;
border-radius:10px;
}

QPushButton:focus
{
color:white;

background:#b20837;
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>Fournisseurs/stock</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/img/img/Untitled design (24).png</normaloff>:/img/img/Untitled design (24).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_employe">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>180</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color:white;
	font: 75 10pt &quot;Source Code Pro&quot;;
	font: 87 8pt &quot;Nexa Rust Sans Black 01&quot;;
    background-color:rgb(39, 39,39);
background: transparent;
border-radius:10px;
}
QPushButton:hover
{
color:white;

background:#b20837;
border-radius:10px;
}

QPushButton:focus
{
color:white;

background:#b20837;
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>Emplois          </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/img/img/Untitled design (18).png</normaloff>:/img/img/Untitled design (18).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_quitter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color:white;
	font: 75 10pt &quot;Source Code Pro&quot;;
	font: 87 8pt &quot;Nexa Rust Sans Black 01&quot;;
    background-color:rgb(39, 39,39);
background: transparent;
border-radius:10px;
}
QPushButton:hover
{
color:white;

background:#b20837;
border-radius:10px;
}

QPushButton:focus
{
color:white;

background:#b20837;
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string> Quitter                    </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/img/img/Untitled design (28).png</normaloff>:/img/img/Untitled design (28).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1256</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
